component/slider/slider.css

```diff
@@ -0,0 +1,10 @@
+.custom-slider{
```

Comment:
i saw that you have used custom-slider className in html doc for styling and for white color you have used value as **white** and at the same time in **volume classname** you have used css variables so its better to use variables everywhere if you have variable for defined color